encoding/ipfix/element.go

```diff
@@ -84,154 +75,154 @@ const (
+func ipfixInformationElements() fieldModel {
+	return fieldModel{
+		elementSpecifier{1, 0}:  fieldDefinition{Uint64, "octetDeltaCount"},
+		elementSpecifier{2, 0}:  fieldDefinition{Uint64, "packetDeltaCount"},
+		elementSpecifier{3, 0}:  fieldDefinition{Uint64, "deltaFlowCount"},
+		elementSpecifier{4, 0}:  fieldDefinition{Uint8, "protocolIdentifier"},
+		elementSpecifier{5, 0}:  fieldDefinition{Uint8, "ipClassOfService"},
+		elementSpecifier{6, 0}:  fieldDefinition{Uint16, "tcpControlBits"},
+		elementSpecifier{7, 0}:  fieldDefinition{Uint16, "sourceTransportPort"},
+		elementSpecifier{8, 0}:  fieldDefinition{IPv4Address, "sourceIPv4Address"},
+		elementSpecifier{9, 0}:  fieldDefinition{Uint8, "sourceIPv4PrefixLength"},
+		elementSpecifier{10, 0}: fieldDefinition{Uint32, "ingressInterface"},
+		elementSpecifier{11, 0}: fieldDefinition{Uint16, "destinationTransportPort"},
+		elementSpecifier{12, 0}: fieldDefinition{IPv4Address, "destinationIPv4Address"},
+		elementSpecifier{13, 0}: fieldDefinition{Uint8, "destinationIPv4PrefixLength"},
+		elementSpecifier{14, 0}: fieldDefinition{Uint32, "egressInterface"},
+		elementSpecifier{15, 0}: fieldDefinition{IPv4Address, "ipNextHopIPv4Address"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{21, 0}: FieldDefinition{Uint32, "flowEndSysUpTime"},
-		ElementSpecifier{22, 0}: FieldDefinition{Uint32, "flowStartSysUpTime"},
+		elementSpecifier{21, 0}: fieldDefinition{Uint32, "flowEndSysUpTime"},
+		elementSpecifier{22, 0}: fieldDefinition{Uint32, "flowStartSysUpTime"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{27, 0}: FieldDefinition{IPv6Address, "sourceIPv6Address"},
-		ElementSpecifier{28, 0}: FieldDefinition{IPv6Address, "destinationIPv6Address"},
-		ElementSpecifier{29, 0}: FieldDefinition{Uint8, "sourceIPv6PrefixLength"},
-		ElementSpecifier{30, 0}: FieldDefinition{Uint8, "destinationIPv6PrefixLength"},
-		ElementSpecifier{31, 0}: FieldDefinition{Uint32, "flowLabelIPv6"},
-		ElementSpecifier{32, 0}: FieldDefinition{Uint16, "icmpTypeCodeIPv4"},
-		ElementSpecifier{33, 0}: FieldDefinition{Uint8, "igmpType"},
+		elementSpecifier{27, 0}: fieldDefinition{IPv6Address, "sourceIPv6Address"},
+		elementSpecifier{28, 0}: fieldDefinition{IPv6Address, "destinationIPv6Address"},
+		elementSpecifier{29, 0}: fieldDefinition{Uint8, "sourceIPv6PrefixLength"},
+		elementSpecifier{30, 0}: fieldDefinition{Uint8, "destinationIPv6PrefixLength"},
+		elementSpecifier{31, 0}: fieldDefinition{Uint32, "flowLabelIPv6"},
+		elementSpecifier{32, 0}: fieldDefinition{Uint16, "icmpTypeCodeIPv4"},
+		elementSpecifier{33, 0}: fieldDefinition{Uint8, "igmpType"},
 		// many, many more, but let's focus on those actually used by my MikroTik
-		ElementSpecifier{56, 0}: FieldDefinition{MacAddress, "sourceMacAddress"},
+		elementSpecifier{56, 0}: fieldDefinition{MacAddress, "sourceMacAddress"},
```

Comment:
Codacy found an issue: [avoid magic numbers like '56', create a named constant for it](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)